### PR TITLE
flow mode reminders and websockets

### PR DESCRIPTION
**Pull request by bjacobson26**
https://www.loom.com/share/de654682459c4c1fb36542b87beed527

### CODE REVIEW

File: lib/managers/ConfigManager.js [hunk 1]
```diff
@@ -54,6 +58,29 @@ export function editPauseSlackNotificationsCallback(value) {
   }
 }
 
+export function editFlowModeReminders() {
+  const configSettings = getConfigSettings();
+  const title = "Flow Mode Reminders";
+  const prompt = "Remind me to enable flow mode when I'm in flow";
```
Comment (brettmstevens7) [inline lib/managers/ConfigManager.js:64]: Are we branding it as "Flow Mode" (uppercase)? @mmclead

**Review by brettmstevens7 — APPROVED** [review]
LGTM!

File: lib/managers/ConfigManager.js (hunk 1 above)
Comment (bjacobson26) [reply lib/managers/ConfigManager.js]: updated

File: lib/websockets.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      initializeWebsockets();
+    }, 10000);
+  });
+
+  ws.on('error', (e) => {
+    console.error("[CodeTime] error connecting to websockets", e);
+  });
+}
+
+export function clearWebsocketConnectionRetryInterval() {
```
Comment (xavluiz) [inline lib/websockets.js:44]: can you call this method from the Activator.js "deactivate()" method?

**Pull request merged**